missing commander dep

```
jade-signer-rpc|feat/openrpc-integration-tests:zap: ⇒ open-rpc-test-coverage -s ./openrpc.json --transport=http --reporter=console 2| pbcopy
internal/modules/cjs/loader.js:628
    throw err;
    ^

Error: Cannot find module 'commander'
Require stack:
- /usr/local/lib/node_modules/@open-rpc/test-coverage/bin/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/usr/local/lib/node_modules/@open-rpc/test-coverage/bin/cli.js:2:17)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:839:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/local/lib/node_modules/@open-rpc/test-coverage/bin/cli.js' ]
}
```

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
     "@types/isomorphic-fetch": "0.0.35",
     "ajv": "^6.10.0",
     "colors": "^1.3.3",
+    "commander": "^2.20.0",
     "isomorphic-fetch": "^2.2.1",
     "json-schema-faker": "^0.5.0-rc16",
     "json-schema-ref-parser": "^6.1.0"

